packages/client/src/forms/utils.ts

```diff
@@ -652,21 +653,16 @@ export const convertToMSISDN = (phone: string) => {
    */
   const countryCode =
     window.config.COUNTRY.toUpperCase() === 'FAR'
-      ? 'ZMB'
-      : window.config.COUNTRY.toUpperCase()
+      ? 'ZM'
+      : window.config.COUNTRY.toUpperCase().slice(0,2)
```

Comment:
I think this will cause bugs. `window.config.COUNTRY` is in alpha-3 format but the phone number lib expects to receive alpha-2 country code. There are cases we it's not as simple as removing the last character when we convert from alpha-3 to alpha-2. Consider these examples:

Sweden's alpha-3 is SWE, but alpha-2 is SE. Similar thing for Andorra and for all of these countries

```
[
    "AND",
    "ABW",
    "BVT",
    "CPV",
    "DNK",
    "SWZ",
    "FLK",
    "HMD",
    "JAM",
    "KAZ",
    "KOR",
    "FSM",
    "PNG",
    "PCN",
    "POL",
    "SPM",
    "SUR",
    "SWE",
    "TUN",
    "TUR",
    "TUV",
    "UKR",
    "UMI",
    "URY",
    "WLF"
]
```

To fix this , use

```
import { callingCountries } from 'country-data'
callingCountries[window.config.COUNTRY.toUpperCase()]
```
and get the alpha-2 country code from there to supply for Google's library